assets/sass/05-blocks/quote/_editor.scss

```diff
@@ -26,36 +59,16 @@
 			line-height: var(--quote--line-height-large);
 		}
 
-		&.has-text-align-right {
-			border-left: none;
-			border-right: var(--quote--border-width) solid var(--quote--border-color);
+		.wp-block-quote__citation {
+			font-size: var(--quote--font-size);
+			line-height: var(--quote--line-height);
 		}
-	}
-
-	&.has-text-align-right {
-		border-right: var(--quote--border-width) solid var(--quote--border-color);
-	}
-
-	&.has-text-align-center {
-		border: none;
-	}
-
-	.has-background:not(.has-background-background-color) &,
-	[class*="background-color"]:not(.has-background-background-color) &,
-	[style*="background-color"]:not(.has-background-background-color) &,
-	.wp-block-cover[style*="background-image"] & {
-		border-color: currentColor;
-	}
 
-	.wp-block-quote__citation {
-		color: var(--global--color-primary);
-		font-size: var(--global--font-size-xs);
-
-		.has-background:not(.has-background-background-color) &,
-		[class*="background-color"]:not(.has-background-background-color) &,
-		[style*="background-color"]:not(.has-background-background-color) &,
-		.wp-block-cover[style*="background-image"] & {
-			color: currentColor;
+		&::before {
+			font-size: 5rem;
+			position: absolute;
+			top: 2px;
+			left: -20px;
```

Comment:
Same note as above with `top: 0;`, and if you make that change these properties can be removed, since it inherits from the above `.wp-block-quote::before`.